README.md

```diff
@@ -177,8 +174,11 @@ PyBaMM is fully open source. For more information about its license, see [LICENS
 Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
 
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+
 <!-- prettier-ignore-start -->
+
 <!-- markdownlint-disable -->
+
```

Comment:
We should not really modify this section (we might end up breaking the bot). The section already has comments in place to ignore the relevant pre-commits. Was this done by one of our pre-commit hooks?

Reply:
these might have done when I ran pre-commit locally without --ignore.
I should have cross checked while commiting, sorry.